java/maven-starter-listen-for-events/src/main/java/com/wgtwo/sample/EventListener.java

```diff
@@ -61,48 +63,48 @@ public void close() {
     private void run() {
         System.out.println("STARTING NEW SUBSCRIPTION");
         // Running in a context to make it possible to cancel it
-        context.run(() -> stub.subscribe(request, new EventObserver()));
+        context.run(() -> stub.streamHandsetChangeEvents(request, new EventObserver()));
 
     }
 
-    public void acknowledge(EventsProto.Event event) {
+    public void acknowledge(AckInfo ackInfo) {
         var request =
-                EventsProto.AckRequest.newBuilder()
-                        .setSequence(event.getMetadata().getSequence())
-                        .setInbox(event.getMetadata().getAckInbox())
+                SubscriptionEventsProto.AckHandsetChangeEventRequest.newBuilder()
+                        .setAckInfo(ackInfo)
                         .build();
```

Comment:
I don't really feel having a specific type for ack request and ack response makes much sense...

Maybe we should just remove that?